test_results/get.go

```diff
@@ -0,0 +1,40 @@
+package test_results
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/evergreen-ci/timber"
+	"github.com/mongodb/grip"
+	"github.com/pkg/errors"
+)
+
+// GetTestResults returns with the test results requested via HTTP to a cedar
+// service.
+func GetTestResults(ctx context.Context, opts timber.GetOptions) ([]byte, error) {
+	if err := opts.Validate(); err != nil {
```

Comment:
The validation for `GetOptions` seems a bit weird on this. If you provide an `ID`, which will not be used for test results but not a `TaskID`, it'll pass validation but won't produce a sensible URL. Can we add logic to ensure that the task ID is set?